.chronus/changes/fix-linting-rule-post-operation-2024-9-15-15-21-46.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-azure-resource-manager"
+---
+
+Fix linting rule to ensure that the linting rule triggers for @get operations on ArmProviderAction
```

Comment:
```suggestion
Fix linting rule to ensure that the linting rule triggers for `@get` operations on ArmProviderAction
```

mark sure to escape decorators or they will tag user with that name in the changelog

Reply:
also message would be good to include the rule name, I changed it but feel free to update to the more accurate description of this fix